src/Native/Runtime/MathHelpers.cpp

```diff
@@ -67,64 +67,103 @@ EXTERN_C REDHAWK_API double REDHAWK_CALLCONV RhpDblRem(double dividend, double d
     // else...
     return(fmod(dividend,divisor));
 }
+
+EXTERN_C REDHAWK_API double REDHAWK_CALLCONV RhpDblRound(double value)
+{
+    return round(value);
+}
+
+EXTERN_C REDHAWK_API float REDHAWK_CALLCONV RhpFltRound(float value)
+{
+    return roundf(value);
+}
+
 #endif // CORERT
 
 #ifdef _ARM_
-EXTERN_C REDHAWK_API Int32 REDHAWK_CALLCONV RhpIDiv(Int32 i, Int32 j)
+EXTERN_C REDHAWK_API Int32 REDHAWK_CALLCONV RhpDiv(Int32 i, Int32 j)
```

Comment:
Could you please keep these as RhpIDiv / RhpIMod; and not delete RhpULMul? 

The .NET Native for UWP depends on these.